coc/raid.py

```diff
@@ -161,15 +155,11 @@ def __repr__(self):
         return "<%s %s>" % (self.__class__.__name__, " ".join("%s=%r" % t for t in attrs),)
 
     def __eq__(self, other):
-        if isinstance(other, RaidAttack):
-            if (self.attacker_tag == other.attacker_tag
+        return (isinstance(other, RaidAttack)
+                and self.attacker_tag == other.attacker_tag
                 and self.destruction == other.destruction
-                and self.stars == other.stars
```

Comment:
Why are we removing the stars from the comparison?

Reply:
because magic and ü think it should be removed. I disagree and because I wrote the code initially it is there and to be removed

Reply:
I wouldn't remove the stars here. It costs us nothing to keep them in (the check is extremely inexpensive) and shuts the door on any potential false positives

Reply:
I mean it should be not possible to get the same destruction with more stars, but you never know🤣

Reply:
I think its not needed, but it also does not matter. If you want to keep it we can keep it